src/modules/SignUp/components/SignUp/components/FormSignUp/FormSignUp.jsx

```diff
@@ -63,7 +63,10 @@ export default class FormSignUp extends Component {
 
         const { formData } = this.state;
 
-        API.signUp(formData)
+        const dataToServer = JSON.parse(JSON.stringify(formData));
```

Comment:
Use [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) for copy an object.